feature-libs/cart/base/components/clear-cart/clear-cart-dialog/clear-cart-dialog.component.html

```diff
@@ -0,0 +1,65 @@
+<ng-container>
```

Comment:
Could we remove <ng-container>?  (unless it serves a purpose)

Reply:
Seems to be removable without consequences